crates/pop-parachains/src/templates.rs

```diff
@@ -123,19 +123,34 @@ pub enum Parachain {
 	EVM,
 	// OpenZeppelin
 	#[strum(
-		serialize = "polkadot-generic-runtime-template",
+		serialize = "generic-template",
```

Comment:
Is this not a breaking change for command line arguments? Will it not affect the analytics?

Reply:
You are right. We'll keep `polkadot-generic-runtime-template` but as deprecated working without showing it to the users `--help`